conf/cassandra-dbaas-light.yaml

```diff
@@ -836,7 +842,9 @@ compaction_throughput_mb_per_sec: 0
 # are completely written, and used in place of the prior sstables for
 # any range that has been written. This helps to smoothly transfer reads
 # between the sstables, reducing page cache churn and keeping hot rows hot
-sstable_preemptive_open_interval_in_mb: 0
+#
+# A negative value disables early open completely
+sstable_preemptive_open_interval_in_mb: -1
```

Comment:
I suspect that setting this value to zero had a considerably negative impact on performance. 

Robert discovered this a long time ago, but we only fixed it in the DSE fork since at the time performance enhancements were not considered for OSS backporting. The details are in [DB-386](https://datastax.jira.com/browse/DB-386).

I would prefer porting this fix to our branch rather than relying on setting `sstable_preemptive_open_interval_in_mb` to -1. When this value is zero, sstables will be opened after every partition, and I doubt that this is what the user intends to do when setting this value to zero.

Reply:
Thanks for the additional details. I considered solving it on the code side (although lacking your context), but opted to set this on the configuration side for backward compatibility. I'll not oppose  if you insist, but I'd rather keep it on the configuration side, and move it back to 0 once we contribute the fix to OSS.

Reply:
As discussed offline, it's fine to set `sstable_preemptive_open_interval_in_mb` to `-1` and port the proper fix to OSS first, but we should ensure no one is bitten by this again, and so I would suggest explaining in the yaml comments how early open works when this value is set to zero.